### PR TITLE
Frontpage rewrite

### PR DESCRIPTION
**Pull request by Be-ing**
Rewrite the front page to not come off as a sensationalist advertisement

### CODE REVIEW

File: pages/index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <li>
       <a href="https://twitter.com/share" class="twitter-share-button"
          data-url="http://www.mixxx.org"
-         data-text="{% blocktrans %}Check out Mixxx - Free MP3 DJ mixing software for Windows, Mac OS X, and Linux{% endblocktrans %}"
+         data-text="{% blocktrans %}Check out Mixxx - Free Digital DJ MP3 mixing software for Windows, Mac OS X, and Linux{% endblocktrans %}"
```
Comment (daschuer) [inline pages/index.html]: "Free" might be an indicator of poor quality. 

How can we make sure that it is "free" in sense of freedom? 
"Free and open source"

Does "digital" add any value? How about remove MP3?

Comment (Be-ing) [inline pages/index.html]: I'm not too worried about being careful about the meaning of "free" in the title text of the web page. I think the rest of the page makes it pretty clear. However, I agree that putting "Free & open source" could help.

I'd like to say just "Digital DJ mixing software", but I think the "MP3" is there for search engine optimization. Mixxx.org appears as the 4th Google search result for "mp3 dj" and the first for "mp3 dj software", so I think that is worth keeping. However, I don't want the title to seem like it is only for MP3s. Also, I think adding "digital" could help Mixxx come up higher in "digital dj", "digital dj software" and related queries.

File: pages/index.html [hunk 2]
```diff
@@ -59,44 +60,43 @@
...
         <img src="{% static '/static/images/squiggle.png' %}">
     </div>-->
-  <p style="text-align: center;">The most advanced free DJ software can be yours in seconds.</p>
+  <p style="text-align: center;">DJ your way. For free.</p>
   <div style="clear: both;"></div>
 </div> <!-- gapfiller -->
 
 <div class="content">
-  <H1>{% trans "Incredible Features, Unbeatable Value" %}</H1>
+  <h1>{% trans "Flexible Features For All DJs" %}</h1>
```
Comment (daschuer) [inline pages/index.html]: What is a "Flexible feature"?  A Feature is never for all DJs.

"Mixxx already covers many use-cases and is open to add your own"
"Mixxx can be tweaked to your needs" 

The "High value for free" is one of the most important property of Mixxx. 

I know users sticking with Virtual DJ just because they have payed for it. 
In German we say: "Was nichts kostet ist auch nichts". We should make sure that this does not apply for Mixxx.

File: pages/index.html [hunk 3]
```diff
@@ -59,44 +60,43 @@
...
 
-  {% blocktrans %}Mixxx has everything you need to start making DJ mixes in a tight, integrated package. Whether you're DJing your next house party, spinning at a club, or broadcasting as a radio DJ, Mixxx has what you need to do it right.{% endblocktrans %}
+  {% blocktrans %}Mixxx has the tools DJs need to perform creative live mixes with digital music files. Whether you are a new DJ with just a laptop or an experienced turntablist, Mixxx can support your style and techniques of mixing.{% endblocktrans %}
```
Comment (daschuer) [inline pages/index.html]: "has the tools" sounds as an independent  tool collection. 

"Mixxx integrates the tools ... "

File: pages/index.html [hunk 4]
```diff
@@ -59,44 +60,43 @@
...
   </div>
 
   <div style="clear: both;"></div>
 
   <div class="halfbox_left">
-    <H2><img src="{% static '/static/images/ic_library_prepare.png' %}" class="feature_icon">{% trans "BPM Detection and Master Sync" %}</H2>
+    <h2><img src="{% static '/static/images/ic_preferences_keydetect.png' %}" class="feature_icon">{% trans "BPM, Key Detection and Sync" %}</h2>
     <p class="feature_indent">
-      {% blocktrans %}Instantly sync the tempo of four songs for seamless beatmixing. Need a break? Create a quick playlist and let Auto DJ take over.{% endblocktrans %}
+      {% blocktrans %}BPM and musical key detection help you find the perfect next track from your library. Use master sync to match the tempo and beats of four songs for seamless mixing.{% endblocktrans %}
```
Comment (daschuer) [inline pages/index.html]: And key ..    harmonic mixing

File: pages/index.html (hunk 3 above)
Comment (ywwg) [inline pages/index.html]: I truly appreciate the enthusiasm but as I stated we are not rewriting the copy at this time.  Please focus your energy elsewhere for now.

File: pages/index.html [hunk 5]
```diff
@@ -59,44 +60,43 @@
...
     </p>
   </div>
 
   <div class="halfbox_right">
-    <H2><img src="{% static '/static/images/ic_preferences_crossfader.png' %}" class="feature_icon">{% trans "Powerful Mixing Engine" %}</H2>
+    <h2><img src="{% static '/static/images/ic_preferences_vinyl.png' %}" class="feature_icon">{% trans "Vinyl Record Control" %}</h2>
     <p class="feature_indent">
-      {% blocktrans %}Mixxx has a cutting-edge mixing engine including support for MP3, M4A/AAC, OGG, FLAC, and WAV audio, adjustable EQ shelves, timecode vinyl control, recording, and Shoutcast broadcasting.{% endblocktrans %}
+      {% blocktrans %}Use turntables with timecode vinyl records to control playback and scratch your digital music files as if they were pressed on vinyl.{% endblocktrans %}
```
Comment (daschuer) [inline pages/index.html]: Path-trough feature .. ?

File: pages/index.html [hunk 6]
```diff
@@ -109,17 +109,13 @@
...
   </p>
 
   <p>
-    {% blocktrans with create_a_branch_link="/get-involved#developers" help_with_translations_link="/get-involved#translators" starter_tasks_link="/get-involved" %}You can get involved with Mixxx today by <a href="{{ create_a_branch_link }}">creating your own branch</a>, <a href="{{ help_with_translations_link }}">helping with translations</a>, or working on one of our other <a href="{{ starter_tasks_link }}">starter tasks</a>.{% endblocktrans %}
+    {% blocktrans with create_a_branch_link="/get-involved#developers" help_with_translations_link="/get-involved#translators" starter_tasks_link="/get-involved" mapping_link="/wiki/doku.php/start#controller_mapping_documentation" %}You can get involved with Mixxx today by <a href="{{ mapping_link }}">making a controller mapping</a>, <a href="{{ create_a_branch_link }}">adding features &amp; fixing bugs</a>, <a href="{{ help_with_translations_link }}">helping with translations</a>, or working on one of our other <a href="{{ starter_tasks_link }}">starter tasks</a>.{% endblocktrans %}
```
Comment (daschuer) [inline pages/index.html]: Test beta versions, file blue-prints and Ideas

File: pages/index.html (hunk 3 above)
Comment (Be-ing) [inline pages/index.html]: > I truly appreciate the enthusiasm but as I stated we are not rewriting the copy at this time. Please focus your energy elsewhere for now.

This has been the last thing on my to-do list before the release and I have arranged my week to get PRs ready with a few days left to review.

File: pages/index.html (hunk 2 above)
Comment (Be-ing) [inline pages/index.html]: I don't mean to imply that every feature is for everyone; I mean to say that "Mixxx already covers many use-cases and is open to add your own" and "Mixxx can be tweaked to your needs". I think the wording "Flexible Features For All DJs" conveys this, but I'll try to come up with some alternatives.

I agree that we should try to counter the stigma against free things implying that they are poor quality. I think the wording proposed does this pretty well. The very first line mentions how Mixxx is free, then this line talks about how Mixxx is quality.

**Comment by daschuer** [comment]
Do we have a not about hardware requirements "netbook approved" ;-)

**Comment by Be-ing** [comment]
> Do we have a not about hardware requirements "netbook approved" ;-) 

No we don't, and I think it should stay that way for this release considering the state of the skins on small screens at the moment. Last I tried running Mixxx on my netbook, Deere was just okay enough to get by with 2 decks and toggling the maximized library. It'd be good to work on them for 2.1 and put it on the website then.

**Comment by daschuer** [comment]
We already have a special netbook skin. It works like a charm for me. 
I think the low hardware requirements is already a big + for Mixxx

But anyway, we should have minimum hardware requirements somewhere.

**Comment by Be-ing** [comment]
> But anyway, we should have minimum hardware requirements somewhere.

This would be nice, but AFAIK no one has really tested what the minimum requirements are for 2.0.

**Comment by daschuer** [comment]
Our minimum reference is my old eeePC 901 with two GB ram and Intel Atom N270 1.6 GHz.

**Comment by daschuer** [comment]
.. running Linux with real time priority. However I have seen Mixxx bitching around on faster machines running Windows.

File: pages/index.html (hunk 1 above)
Comment (Be-ing) [inline pages/index.html]: I tried making the title "Free & Open Source Digital DJ MP3 Mixing Software", but that feels way too wordy and complicated. I think it should be left as "Free Digital DJ MP3 Mixing Software".

File: pages/index.html (hunk 6 above)
Comment (Be-ing) [inline pages/index.html]: When we release the next beta, I think that would be good to put here. For now, I think it's better without that. I added a link to Launchpad.

**Comment by Be-ing** [comment]
If there is a place to mention netbooks, it's not the frontpage.

**Comment by Be-ing** [comment]
@esbrandt any other thoughts on this?

**Comment by Be-ing** [comment]
@ywwg want to take a look at this?

File: pages/index.html [hunk 7]
```diff
@@ -60,44 +54,43 @@ <H1>{% trans "DJ Your Way. For Free." %}</H1>
     <div style="float: right; margin-top: 16px; padding-right: 8px;">
         <img src="{% static '/static/images/squiggle.png' %}">
     </div>-->
-  <p style="text-align: center;">The most advanced free DJ software can be yours in seconds.</p>
+  <p style="text-align: center;">Free and open source DJ software for Windows, Mac OS X, and Linux</p>
```
Comment (rryan) [inline pages/index.html]: Update to macOS?

**Comment by Be-ing** [comment]
Ready for merge?

**Comment by Be-ing** [comment]
Okay, I took a new screenshot with Deere playing 4 decks. I don't know how to make it look all fancy like the old screenshots, so it's just a plain screenshot for now. If anyone wants to improve it, go for it, but I think this is fine for the initial release announcement. The original image is in assets/screenshot_sources.

**Comment by ywwg** [comment]
> I don't know how to make it look all fancy like the old screenshots,

I use gimp and draw an ellipse, invert it so it's selecting outside, then feather a couple hundred pixels.  Then I fill the selection with black and that darkens the borders.  I usually use a separate layer so I can adjust opacity for more or less darkening.

(http://ywwg.com/images/deere-splash.png)

File: pages/index.html [hunk 8]
```diff
@@ -2,15 +2,15 @@
 
 {% load i18n %}
 
-{% block title %}{% trans "Free MP3 DJ Mixing Software" %}{% endblock %}
+{% block title %}{% trans "Free Digital DJ MP3 Mixing Software" %}{% endblock %}
```
Comment (ywwg) [inline pages/index.html:5]: maybe we can drop "mp3" now?  It seems sort of quaint at this point.  (optional)

File: pages/index.html [hunk 9]
```diff
@@ -60,44 +54,43 @@ <H1>{% trans "DJ Your Way. For Free." %}</H1>
...
+      {% blocktrans %}BPM and musical key detection help you find the perfect next track from your library. Use master sync to match the tempo and beats of four songs for seamless mixing.{% endblocktrans %}
     </p>
   </div>
 
   <div class="halfbox_right">
-    <H2><img src="{% static '/static/images/ic_preferences_controllers.png' %}" class="feature_icon">{% trans "DJ Controller Support" %}</H2>
+    <h2><img src="{% static '/static/images/ic_preferences_controllers.png' %}" class="feature_icon">{% trans "DJ Controller Support" %}</h2>
     <p class="feature_indent">
-      {% blocktrans %}With over 85 MIDI DJ controllers supported out-of-the-box and several HID controllers, Mixxx gives you comprehensive hardware control for your DJ mixes.{% endblocktrans %}
+      {% blocktrans %}Built-in mappings for DJ controller hardware gives you hands-on control of Mixxx's features. Use the programmable mapping system to customize your workflow and add support for any MIDI or HID device.{% endblocktrans %}
```
Comment (ywwg) [inline pages/index.html:76]: "Built-in and supported mappings for many popular DJ controllers"  ?

File: pages/index.html [hunk 10]
```diff
@@ -60,44 +54,43 @@ <H1>{% trans "DJ Your Way. For Free." %}</H1>
...
     </p>
   </div>
 
   <div style="clear: both;"></div>
 
   <div class="halfbox_left">
-    <H2><img src="{% static '/static/images/ic_library_prepare.png' %}" class="feature_icon">{% trans "BPM Detection and Master Sync" %}</H2>
+    <h2><img src="{% static '/static/images/ic_preferences_waveform.png' %}" class="feature_icon">{% trans "Effects" %}</h2>
     <p class="feature_indent">
-      {% blocktrans %}Instantly sync the tempo of four songs for seamless beatmixing. Need a break? Create a quick playlist and let Auto DJ take over.{% endblocktrans %}
```
Comment (ywwg) [inline pages/index.html:91]: autodj is a very popular feature with our users, we shouldn't drop it.

**Review by ywwg — COMMENTED** [review]
I still prefer a slightly more excited voice in our marketing, but this is fine.

I notice we don't mention streaming / shoutcast support on the front page.  We have a *lot* of podcasters and even internet radio stations that use mixxx, we should call attention to those features more.

File: pages/index.html (hunk 8 above)
Comment (Be-ing) [reply pages/index.html:5]: As discussed before, it's helpful for search engines. Even though it is a little misleading because Mixxx doesn't play only MP3s, that's what a lot of people are searching for.

File: pages/index.html (hunk 9 above)
Comment (Be-ing) [reply pages/index.html:76]: I think it's fine how it is. "Supported" may be a bit too strong of a word when many mappings are contributed by people who do not stick around.

File: pages/index.html (hunk 10 above)
Comment (Be-ing) [reply pages/index.html:91]: Eh, I don't see an easy way to fit it into the 2 x 2 grid of highlighted core features...

**Comment by Be-ing** [comment]
> I use gimp and draw an ellipse, invert it so it's selecting outside, then feather a couple hundred pixels. Then I fill the selection with black and that darkens the borders. I usually use a separate layer so I can adjust opacity for more or less darkening.

Great, thanks for doing that!
